romana/client/client.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
Maybe adaptor? Even though that name might be reserved for an actual stand alone service. I think there is definitely a place for such a thing. We don't have it yet, though. Eventually (once or if we have it), the command line tools might communicate with that adaptor service, so the code dealing with different environments would move into the adaptor service and out of the command line tools.

So, having it here may be temporary anyway. The adaptor name therefore could be used here, for the time being. Maybe call it adaptorlib or something.